frontend/src/pages/Interventions/data/actions.ts

```diff
@@ -0,0 +1,20 @@
+import {ActionTypes, Action, Interventions} from "../../../models/types";
+
+interface GetInterventionsAction {
+  error?: string,
+  interventions?: Interventions,
+}
```

Comment:
I think we should split this up into an `Error` type and an `Intervention` type

Reply:
I personally think it is okay since all the request, error, and success stuff belongs to this GetInterventionsAction part